_posts/2019/2019-02-04-napalm-inspector-launched.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Napalm-inspector 1.0.0 Released"
+author: ogenstad
+description: "Napalm-inspector is a web application aimed to help troubleshoot the Napalm getters"
+tags: ["news"]
+---
+I wish I could say that you should be excited about Napalm-inspector. Unfortunately, very little unicorn SDN is involved in this new tool it's actually pretty dull. It's not the type of application that will propel you to the next level in your network automation journey and allow you to crush it. Instead, Napalm-inspector is a web application aimed to help with the work on bug fixes and troubleshooting around the Napalm getters. 
+<!--more-->
+### Making sense of unstructured data
+One of the goals of Napalm is to provide its users with a unified API which can return structured data which follows the same format regardless of device type. For example, if you use the [get_bgp_neighbor()](https://napalm.readthedocs.io/en/latest/base.html#napalm.base.base.NetworkDriver.get_bgp_neighbors) function the data will look the same regardless if you target a Cisco IOS device, a Cisco IOS XR one or a Junos router. For some of the platforms that Napalm supports the data is already in a structured format, and all Napalm has to do is to render the data in the format that Napalm uses.
+
+However, for some devices, the data is entirely unstructured, so Napalm needs to parse the output from these platforms to make sense of the data. For devices in this category, Napalm sends show commands to them and feeds the output through the parsing engine. For each getter, Napalm runs one or more commands and analyzes the output. In a perfect world, this would be done once for each getter function, and everything would work. In our world this is where the unicorn starts to cry. Sadly the output for a given command often looks different between Cisco IOS versions, in some cases, the commands aren't even the same.
```

Comment:
> Sadly the output for a given command often looks different between Cisco IOS versions, in some cases, the commands aren't even the same.

I'd rephrase to something more generic, ie:

Sadly the output for a given command differs between versions of the same NOS and, in some cases, even different commands are needed to gather the needed information.